src/jloptions.c

```diff
@@ -161,8 +161,8 @@ static const char opts_hidden[]  =
     " --output-bc name          Generate LLVM bitcode (.bc)\n"
     " --output-asm name         Generate an assembly file (.s)\n"
     " --output-incremental=no   Generate an incremental output file (rather than complete)\n"
-    " --trace-compile={stdout,stderr}\n"
-    "                           Print precompile statements for methods compiled during execution.\n\n"
+    " --trace-compile={stdout,stderr,name}\n"
+    "                           Print precompile statements for methods compiled during execution or save to a file (.jl)\n\n"
```

Comment:
```suggestion
    "                           Print precompile statements for methods compiled during execution or save to a path\n\n"
```
Otherwise it makes it seem like this flag will write to the file `$name.jl` which is not what it should do.

Reply:
I changed the word to just "path" as suggested.